library/src/main/java/com/schibsted/spain/barista/cleardata/ClearFilesRule.java

```diff
@@ -0,0 +1,33 @@
+package com.schibsted.spain.barista.cleardata;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/**
+ * This rule clears all app's stored files before running each test
+ */
+public class ClearFilesRule implements TestRule {
+
+  @Override
+  public Statement apply(final Statement base, Description description) {
+    return new Statement() {
+      @Override
+      public void evaluate() throws Throwable {
+        clearData();
+        base.evaluate();
+        clearData();
+      }
+    };
+  }
+
+  private void clearData() {
+    FileOperations.deleteAllFiles(getAppContext());
```

Comment:
Just for being lean, I'm trying to do the smaller approach that fixes the issue we had yesterday at Fotocasa. If other apps need other approaches, we can do it on demand. But I feel that right now, a Rule that justs remove all the files is good enough...